test/new-e2e/utils/infra/stack_manager.go

```diff
@@ -119,10 +119,31 @@ func (sm *StackManager) GetStack(ctx context.Context, name string, config runner
 }
 
 func (sm *StackManager) DeleteStack(ctx context.Context, name string) error {
+	var stack *auto.Stack
+	var ok bool
+
 	sm.lock.Lock()
 	defer sm.lock.Unlock()
 
-	return sm.deleteStack(ctx, name, sm.stacks[name])
+	stack, ok = sm.stacks[name]
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Minor: you should not need to init these at lines 122/123, you can init at assignment with `:=`